src/backend/access/external/plugstorage.c

```diff
@@ -89,6 +94,11 @@ void getExternalTableTypeInStr(const char formatType,
 		*formatterName = NULL;
 	}
 	else if (fmttype_is_custom(formatType))
+	{
+		*formatterType = ExternalTableType_CUSTOM;
+		*formatterName = NULL;
+	}
+	else if (fmttype_is_custom(formatType))
```

Comment:
There is a same if condition above (`fmttype_is_custom(formatType)`).
Is one of them useless?

Reply:
Yes. The latter branch is disabled temporarily, serving as a workaround to fix the PXF issue. Once the pluggable storage framework is well tested, this workaround will be dropped.

Reply:
Thanks!